### ISSUE TITLE
RC Test: Least Squares Regression 1.1.0-rc.1

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#542]
Least Squares Regression 1.1.0-rc.1 is ready for testing.

**Link to Sim:** http://www.colorado.edu/physics/phet/dev/html/least-squares-regression/1.1.0-rc.1/least-squares-regression_en.html						

**Line to iFrame:** http://www.colorado.edu/physics/phet/dev/html/least-squares-regression/1.1.0-rc.1/least-squares-regression_en-iframe.html

**Link to test matrix:** https://docs.google.com/spreadsheets/d/1fuKgDgWtNw0QjrkA08F4albsVDKlmdwjTjop5qszBi4/edit#gid=762325686

Please do a full test of this RC on all platforms, and run the following to check internationalization:

- [x] stringTest=double
- [x] stringTest=long
- [x] stringTest=x
- [x] stringTest=rtl
- [x] stringTest=xss (should not redirect, OK to look bad, test on one desktop platform)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test
- [x] Reference issues to RC test task

The purpose for this release is making the sim translatable, and we are not resolving all known issues in this release.
If you notice an issue that is not already logged in https://github.com/phetsims/least-squares-regression/issues, please create a new issue and reference this one.

**Comment by ariel-phet** [comment]
@BryceAG @brroberts1231 @elisemorgan @bryo5363 @phet-steele please makes this the t next testing priority after #541 is complete.

**Comment by phet-steele** [comment]
@jessegreenberg @ariel-phet done.

**Comment by jessegreenberg** [comment]
Thank you @phet-steele!  I will have a new RC out soon, closing this issue.
